core/init.go

```diff
@@ -176,7 +178,7 @@ func initKongRoutes(url string, c *http.Client, r *KongRoute, path string, name
 		if resp.StatusCode == 200 || resp.StatusCode == 201 || resp.StatusCode == 409 {
 			lc.Info(fmt.Sprintf("Successful to set up route for %s.", name))
 		} else {
-			s := fmt.Sprintf("Failed to set up route for %s with errorcode %d.", name, resp.StatusCode)
+			s := fmt.Sprintf("Failed to set up route for %s with errore %s.", name, resp.Status)
```

Comment:
Typo, `errore` should be `error`

Reply:
get another push to fix the typo.

Reply:
and the PR should remove binary file under core.